activerecord/test/cases/adapters/postgresql/comment_test.rb

```diff
@@ -0,0 +1,87 @@
+require 'cases/helper'
+require 'support/schema_dumping_helper'
+
+class PostgresqlCommentTest < ActiveRecord::PostgreSQLTestCase
+  include SchemaDumpingHelper
+
+  class PgComment < ActiveRecord::Base
+    self.table_name = 'pg_comments'
+  end
+
+  def setup
+    @connection = ActiveRecord::Base.connection
+
+    @connection.transaction do
+      @table = @connection.create_table('pg_comments', comment: 'A table with comment', force: true) do |t|
+        t.string  'name',    comment: 'Comment should help clarify the column purpose'
+        t.boolean 'obvious', comment: 'Question is: should you comment obviously named objects?'
+        t.string  'content'
+        t.index   'name',    comment: 'Very important index that powers all the performance'
```

Comment:
I suggest to include a single quote `'`, a double quote `"`, and a new line `\n` characters in test comments to validate that there are no issues with correct quoting and producing schema dumps.